demisto_sdk/commands/common/hook_validations/id.py

```diff
@@ -212,6 +215,33 @@ def _is_mapper_incident_types_found(self, mapper_data):
 
         return is_valid
 
+    def _is_pack_display_name_already_exist(self, pack_metadata_data):
+        """Check if the pack display name already exists
+        Args:
+            pack_metadata_data (dict): Dictionary that holds the extracted details from the given metadata file.
+        Returns:
+            bool. Whether the metadata file is valid or not.
+        """
+        new_pack_id = list(pack_metadata_data.keys())[0]
+        new_pack_name = pack_metadata_data[new_pack_id]['name']
+        for pack_id, pack_data in self.packs_set.items():
+            if new_pack_name == pack_data['name'] and new_pack_id != pack_id:
+                return False, Errors.pack_name_already_exists(new_pack_name)
+        return True, None
+
+    def is_unique_file_valid_in_set(self, pack_path, ignored_errors=None):
+        self.ignored_errors = ignored_errors
+        is_valid = True
+        self.is_circle = True
+        error = None
+        if self.is_circle:  # No need to check on local env because the id_set will contain this info after the commit
```

Comment:
`self.is_circle` is true (hardcoded), then we check if it's true. is that by design?
Also, not sure I understand the comment

Reply:
done